### PR TITLE
Further changes to parser arguments

### PR DESCRIPTION
**Pull request by MichaelLLi**

### CODE REVIEW

**Review by hamzatazib — CHANGES_REQUESTED** [review]
DELPHI_utils_V3_static.py: I had renamed fitcasesnd and fitcasesd things into cases_fit_data and deaths_fit_data for clarity for other people, can you please bring it back ? :)

**Comment by MichaelLLi** [comment]
Done!

File: DELPHI_model_V3.py [hunk 1]
```diff
@@ -446,7 +457,7 @@ def solve_best_params_and_predict(optimal_params):
     logging.info(f"Number of CPUs found and used in this run: {n_cpu}")
 
     list_tuples = popcountries.tuple_area.tolist()
-#    list_tuples = [x for x in list_tuples if x[0] == "Oceania"]
+    list_tuples = [x for x in list_tuples if x[0] == "Oceania"]
```
Comment (sakshamsoni97) [inline DELPHI_model_V3.py:460]: I think this should be commented out like it was before.

Comment (hamzatazib) [inline DELPHI_model_V3.py:460]: yup comment this out plz

File: DELPHI_model_V3.py [hunk 2]
```diff
@@ -74,15 +74,28 @@
...
-    help="Generate Confidence Intervals? Reply 0 or 1 (for False or True).",
+    '--confidence_intervals', '-ci', type=bool, required=True, choices=[False, True],
+    help="Generate Confidence Intervals? Reply False or True.",
+)
+
+parser.add_argument(
+    '--since100case', '-s100', type=bool, required=True, choices=[False, True],
+    help="Save all history (since 100 cases)? Reply False or True.",
+)
+
```
Comment (hamzatazib) [inline DELPHI_model_V3.py:85]: delete this extra space

**Review by hamzatazib — CHANGES_REQUESTED** [review]

File: DELPHI_model_V3.py (hunk 1 above)
Comment (MichaelLLi) [reply DELPHI_model_V3.py:460]: LOL but I only want to do Oceania

Comment (hamzatazib) [reply DELPHI_model_V3.py:460]: This is gonna be merged into master, no can do :(

File: DELPHI_model_V3.py (hunk 2 above)
Comment (MichaelLLi) [reply DELPHI_model_V3.py]: Done

Comment (hamzatazib) [reply DELPHI_model_V3.py]: Think you haven't pushed the one where you delete the extra space in line 80

**Review by hamzatazib — APPROVED** [review]

**Pull request merged**